android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -111,39 +120,44 @@ public IBinder onBind(Intent intent) {
 
     @Override
     public int onStartCommand(Intent intent, int flags, int startId) {
-        tryParseIsAudioMuted(intent);
+
+        Boolean isAudioMuted = tryParseIsAudioMuted(intent);
+
+        if (isAudioMuted != null) {
+            this.isAudioMuted = Boolean.parseBoolean(intent.getStringExtra("muted"));
+
+            Notification notification = OngoingNotification.buildOngoingConferenceNotification(isAudioMuted);
+            if (notification == null) {
+                stopSelf();
+                JitsiMeetLogger.w(TAG + " Couldn't start service, notification is null");
+            } else {
+                NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+                notificationManager.notify(OngoingNotification.NOTIFICATION_ID, notification);
+            }
+        }
 
         final String actionName = intent.getAction();
         final Action action = Action.fromName(actionName);
 
-        switch (action) {
-            case UNMUTE:
-            case MUTE:
-                Intent muteBroadcastIntent = BroadcastIntentHelper.buildSetAudioMutedIntent(action == Action.MUTE);
-                LocalBroadcastManager.getInstance(getApplicationContext()).sendBroadcast(muteBroadcastIntent);
-                break;
-            case START:
-                Notification notification = OngoingNotification.buildOngoingConferenceNotification(isAudioMuted);
-                if (notification == null) {
-                    stopSelf();
-                    JitsiMeetLogger.w(TAG + " Couldn't start service, notification is null");
-                } else {
-                    startForeground(OngoingNotification.NOTIFICATION_ID, notification);
-                    JitsiMeetLogger.i(TAG + " Service started");
-                }
-                break;
-            case HANGUP:
-                JitsiMeetLogger.i(TAG + " Hangup requested");
+        if (action != null) {
```

Comment:
how can action be null?

Reply:
The action is null when there is no action name passed in the Intent (or one that does not match the 3 actions we got). When starting the service, there isn't actually any name in the intent, so this is null

Reply:
Gotcha, can you please leave a comment? I know I'll forget this :-P